test/specs/home.spec.js

```diff
@@ -0,0 +1,32 @@
+import HomePage from '../pageobjects/home.page.js'
+import LocationPopUp from '../pageobjects/locationPopUp.page.js'
+import SignInPopUp from '../pageobjects/singInPopUp.page.js'
+import fs from 'fs'
+
+describe('Assertion for Amazom Home Page', async () => {
+    it('verify Amazon Logo', async () => {
+        const testData = await JSON.parse(fs.readFileSync('test/testData.json'))
+        await HomePage.open()
+        await expect(HomePage.logo).toExist()
+        await expect(HomePage.logo).toHaveAttribute('aria-label')
+        await expect(HomePage.logo).toHaveAttributeContaining('aria-label', testData.brand)
+    })
+
+    it('verify Delivery Location', async () => {
+        await HomePage.open()
+        await expect(HomePage.actualDeliveryLocation).toHaveTextContaining('Belarus')
```

Comment:
you have 'Belarus' in the file testData.json

Reply:
fixed